.github/workflows/balena.yml

```diff
@@ -32,8 +32,9 @@ env:
     export-database \
     import-database \
     e2e-server-previous-dump
-
-  LIVECHAT_PORT: 443
+  INTEGRATION_JELLYFISH_APP_ID: jellyfish
+  INTEGRATION_JELLYFISH_APP_SECRET: changeme
+  HYDRA_ADMIN_HOST: hydra:4445
```

Comment:
@karaxuna Would this be better as `http://hydra:4445`? Found that `nock` throws an error when you only set `hydra:4445`, seemingly requiring the protocol be provided. Currently covering this in `integration-server` tests by setting `HYDRA_ADMIN_HOST=http://hydra:4445` in `apps/server/package.json` to override the value given here.

Reply:
I still got same error. Can't understand why request does not get mocked.

Reply:
The `integration-server` tests are passing, now its getting stuck at `e2e-ui` when attempting to log in as the test user with the sdk.